enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/InternalFlatFileRealmIT.java

```diff
@@ -91,9 +91,10 @@ public void setup() throws Throwable
     }
 
     @After
-    public void teardown() throws Throwable
+    public void tearDown() throws Throwable
     {
         realm.shutdown();
+        fs.close();
```

Comment:
`fs` here is an `EvilFileSystem` wrapping an `EphemeralFileSystem` created by a rule. I think the rule should be removed as this is the only place it is used.